guides.md

```diff
@@ -19,6 +19,7 @@ our values.
 
 - [Meetings](#meetings)
 - [Shortlinks](#shortlinks)
+- [Storage: Sensitive Data](#storage-sensitive-data)
```

Comment:
Maybe the `Storage: ` makes the index looks confusing?

```suggestion
- [Sensitive Data](#storage-sensitive-data)
```

Reply:
```suggestion
- [Sensitive Data](#sensitive-data)
```